docs/interpreter/jdbc.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: page
+title: "Generic JDBC Interpreter"
+description: "JDBC user guide"
+group: manual
+---
+{% include JB/setup %}
+
+
+## Generic JDBC  Interpreter for Apache Zeppelin
+
+This interpreter lets you create a JDBC connection to any data source, by now it has been tested with:
+
+* Postgres
+* MySql
+* MariaDB
+* Redshift
+* Hive
+
+If someone else used another database please report how it works to improve functionality.
+
+### Create Interpreter
+
+#### Simple connection
+
+Before creating the interpreter it is necessary to add to the Zeppelin classpath the path of the JDBC you want to use, to do it you must edit the file `zeppelin-daemon.sh` as shown in the image:
```

Comment:
oh sorry, I've missed this - please update the text `as shown in the image` since there is no image now?